website/content/api-docs/system/inspect/index.mdx

```diff
@@ -0,0 +1,17 @@
+---
+layout: api
+page_title: /sys/internal/inspect - HTTP API
```

Comment:
The RFC mentions that because this would expose information at a similar sensitivity to `sys/raw`, there would be more required to enable the endpoint. We should document somewhere how to enable and disable the feature. `sys/raw` config is documented here https://developer.hashicorp.com/vault/docs/configuration#parameters and the API docs (https://developer.hashicorp.com/vault/api-docs/system/raw) link to that page.

Reply:
Ah yes - added this as well.

Reply:
Thanks for adding the link, but I think we also should document what they need to change in the configuration to enable this?